data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/BeaconStateHandler.java

```diff
@@ -69,22 +85,39 @@ private BeaconState queryByRootHash(String root) {
             description = "The beacon state matching the supplied query parameter was not found."),
         @OpenApiResponse(status = "400", description = "Missing a query parameter")
       })
+  private String validateParams(final Map<String, List<String>> params, final String key) {
```

Comment:
I'm surprised putting the `@OpenApi` on this private method works. Shouldn't it be on the `handle` method?

Reply:
that's just me being an idiot... it's not the handle method. it was a late change to centralise the validation...